src/cmds/scripts/pbs_habitat.in

```diff
@@ -514,7 +514,15 @@ if [ -z "$PBS_HOME" ]; then
 	echo "*** PBS_HOME is not set"
 	exit 1
 fi
+
+update_home=0
 if [ ! -d "$PBS_HOME" ]; then
+        update_home=1
+else
+	# Check if PBS_HOME is empty.
+        [ ! "$(/bin/ls -A $PBS_HOME)" ] && update_home=1
```

Comment:
How about you put this one in above if condition? like `if [ ! -d "${PBS_HOME}" -o ! -z "$(/bin/ls -A ${PBS_HOME})" ]; then`

Reply:
I thought of it but then just the sake of readability I broke the code. If you insist on merging the conditions I can do that :)

Reply:
yes please merge it, about readability I think merged if will look more readable then current one

Reply:
Done. Please check now.